main/streams/glob_wrapper.c

```diff
@@ -79,6 +82,9 @@ PHPAPI char* _php_glob_stream_get_pattern(php_stream *stream, size_t *plen STREA
 }
 /* }}} */
 
+#define _php_glob_stream_get_result_count(pglob) \
+	pglob->open_basedir_indexmap_used ? (int) pglob->open_basedir_indexmap_size : pglob->glob.gl_pathc
```

Comment:
Since this is a compilation unit internal macro, it is okay as is, but it would be more safe like:
```suggestion
#define _php_glob_stream_get_result_count(pglob) \
	do {(pglob)->open_basedir_indexmap_used ? (int) (pglob)->open_basedir_indexmap_size : (pglob)->glob.gl_pathc;} while (0)
```
Or maybe use a static inline function to avoid any issues (especially double evaluation of the argument).

Reply:
Yeah static inline function is better here...